proto/ibc/applications/transfer/v1/genesis.proto

```diff
@@ -5,11 +5,16 @@ package ibc.applications.transfer.v1;
 option go_package = "github.com/cosmos/ibc-go/v7/modules/apps/transfer/types";
 
 import "ibc/applications/transfer/v1/transfer.proto";
+import "cosmos/base/v1beta1/coin.proto";
 import "gogoproto/gogo.proto";
 
 // GenesisState defines the ibc-transfer genesis state
 message GenesisState {
   string              port_id      = 1;
   repeated DenomTrace denom_traces = 2 [(gogoproto.castrepeated) = "Traces", (gogoproto.nullable) = false];
   Params              params       = 3 [(gogoproto.nullable) = false];
+  // denom_escrows contains the denomination and the total amount of tokens of
+  // the denomination in escrow in the transfer channels accounts
+  repeated cosmos.base.v1beta1.Coin denom_escrows = 4
```

Comment:
Should we call this `total_escrowed`?

Reply:
And `denom_total_escrow`? I was trying to keep it similar to `denom_traces` above...

Reply:
I personally don't think `denom_traces` is a good name. I don't think we need to keep to that pattern, but happy to abide by what folks think is best